### PR TITLE
Fix/add queue handler

### PR DESCRIPTION
**Pull request by chunfantasy**
This PR is due to fixing race condition while handling log messages and status.

### CODE REVIEW

File: pc_ble_driver_py/ble_driver.py [hunk 1]
```diff
@@ -43,9 +43,10 @@
 import sys
 import time
 import traceback
+import queue
 import logging
 from abc import abstractmethod
-from threading import Lock
+from threading import Thread, Lock
```
Comment (bihanssen) [inline pc_ble_driver_py/ble_driver.py]: Is Lock module import needed?

File: pc_ble_driver_py/ble_driver.py [hunk 2]
```diff
@@ -1637,12 +1644,23 @@ def status_handler_sync(self, adapter, status_code, status_message):
...
 
+    def status_handler_thread(self):
+        while True:
+            try:
+                item = self.status_queue.get()
+                if item is None:
+                    continue
+                self.status_handler_sync(*item)
+            except Exception as ex:
+                print('Exception in status handler: {}'.format(ex))
```
Comment (bihanssen) [inline pc_ble_driver_py/ble_driver.py]: Could replace `print` with `logger.exception`. Same on line 1692.

File: pc_ble_driver_py/ble_driver.py (hunk 1 above)
Comment (chunfantasy) [reply pc_ble_driver_py/ble_driver.py]: Yes, since observer_lock and api_lock are still in use.

File: pc_ble_driver_py/ble_driver.py (hunk 2 above)
Comment (chunfantasy) [reply pc_ble_driver_py/ble_driver.py]: Yes.

**Pull request merged**